### DISCUSSION TITLE
GH Organisation level "Action Secrets" // in which plan?

### DISCUSSION THREAD

**Discussion by daniel-ciaglia** [community/community#33712] [Actions] [answered]
Hi,
I can not find the answer to this, but at which plan do I get organisation level "GH Action Secrets"?

> Organization secrets cannot be used by private repositories with your plan.
> Please consider [upgrading your plan](https://github.com/organizations/mindmonopol/settings/billing) if you require this functionality.

- GH Free for organisations -> asks for the above plan upgrade
- GH Team?
- GH Enterprise?

It is not clear to me from the documentation: https://docs.github.com/en/get-started/learning-about-github/githubs-products#github-team

Regards, Daniel

**Comment by daniel-ciaglia** [comment, marked as answer]
The org wide secrets are available from Team plan on (according to GH Support).

**Reply by victor-tseng-ttfb** [reply]
In GitHub doc: https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-an-organization when you select plan as ”Free, Pro & Team“ it shows not available to your plan. And if you switch to Enterprise Cloud this warning is gone.

I wonder if the pricing plan has changed? And how do I reach GH Support if I am currently in Free plan?

**Reply by victor-tseng-ttfb** [reply]
I just actually tried to buy Team plan with my testing 1 seat org and confirmed that it allows org level secret.

**Reply by hugebug4ever** [reply]
> I just actually tried to buy Team plan with my testing 1 seat org and confirmed that it allows org level secret.

I tried too, it do works for me as well.
